mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule ExMQTT.MixProject do
   def project do
     [
       app: :exmqtt,
-      version: "0.1.0",
+      version: "1.0.0",
```

Comment:
I don't feel comfortable with `1.0.0` yet since I haven't used it enough. Do you use this in production?

`0.2.0` would probably be better for now

Reply:
I don't use it in production yet. 
It's ok I'll change the version to 0.2.0!